### PR TITLE
Optimize repo-list layout to enhance visual experience

### PR DESCRIPTION
**Pull request by kerwin612**
before: 

***The problem was that the icon and text were not on a horizontal line, and the horizontal was not centered;***

after:

### CODE REVIEW

File: templates/user/settings/repos.tmpl [hunk 1]
```diff
@@ -84,17 +84,17 @@
 					<div class="ui middle aligned divided list">
 						{{range .Repos}}
 							<div class="item">
-								<div class="content">
+								<div class="tw-flex tw-items-center tw-flex-wrap tw-gap-1 content">
```
Comment (wxiaoguang) [inline templates/user/settings/repos.tmpl]:
```suggestion
								<div class="flex-text-block content">
```

Not sure whether `tw-flex-wrap` is right, but I guess no.

Comment (kerwin612) [reply templates/user/settings/repos.tmpl]: There isn't a need for `tw-flex-wrap` here, I removed it;

File: templates/user/settings/repos.tmpl [hunk 2]
```diff
@@ -84,17 +84,17 @@
 					<div class="ui middle aligned divided list">
 						{{range .Repos}}
 							<div class="item">
-								<div class="content">
+								<div class="tw-flex tw-items-center tw-gap-1 content">
```
Comment (wxiaoguang) [inline templates/user/settings/repos.tmpl]:
```suggestion
								<div class="content flex-text-block">
```

**Comment by wxiaoguang** [comment]
One more thing, I think we should remove all "iconFloat". It can't be right with "flex".

image

**Review by delvh — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Pull request merged**